enterprise/server/tasksize/tasksize.go

```diff
@@ -322,7 +329,16 @@ func testSize(testSize string) (int64, int64) {
 // from the task such as test size and estimated compute units, but does not use
 // information about historical task executions.
 func Estimate(task *repb.ExecutionTask) *scpb.TaskSize {
-	props := platform.ParseProperties(task)
+	props, err := platform.ParseProperties(task)
+	if err != nil {
+		log.Warningf("Failed to parse task properties, using default estimation: %s", err)
+
+		return applyMinimums(&scpb.TaskSize{
+			EstimatedMemoryBytes:   DefaultMemEstimate,
+			EstimatedMilliCpu:      DefaultCPUEstimate,
+			EstimatedFreeDiskBytes: DefaultFreeDiskEstimate,
+		})
```

Comment:
This skips the "less than Max value" check and relies on the default value to always be less than Max.